config/languages.js

```diff
@@ -10,6 +10,11 @@ module.exports = {
       shortName: 'Fra',
       code: 'fr-fr',
     },
+    pt: {
+      name: 'Portuguesa',
```

Comment:
I think here should be 
```suggestion
      name: 'Português',
```